client/src/App.js

```diff
@@ -6,21 +6,38 @@ import theme from './theme';
 import Start from '../src/pages/Start';
 import Login from '../src/pages/Login';
 
+const [test, setTest] = React.useState(null);
+
+React.useEffect(() => {
+  fetch('/api/test')
+    .then((response) => response.json())
+    .then((result) => {
+      setTest(result);
+    });
+}, []);
+
 function App() {
   return (
-    <ThemeProvider theme={theme}>
-      <GlobalStyles />
-      <Router>
-        <Switch>
-          <Route exact path="/">
-            <Start />
-          </Route>
-          <Route exact path="/login">
-            <Login />
-          </Route>
-        </Switch>
-      </Router>
-    </ThemeProvider>
+    <>
+      {test && (
```

Comment:
If you test something and want feedback, make it a draft-request. If you merge this pull-request you will have test-stuff in your master branch, which you need to remove later.